From 3a22b4650ead54a28315e3c0e5116ca5e130e977 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Mon, 31 Aug 2020 16:35:28 +0200
Subject: [PATCH] char: virtio: Select VIRTIO from VIRTIO_CONSOLE.

References: bsc#1175667
Patch-mainline: submitted https://lore.kernel.org/lkml/20200831165850.26163-1-msuchanek@suse.de/

Make it possible to make virtio console built-in when
other virtio drivers are modular.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 drivers/char/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/char/Kconfig b/drivers/char/Kconfig
--- a/drivers/char/Kconfig
+++ b/drivers/char/Kconfig
@@ -158,8 +158,9 @@ source "drivers/tty/hvc/Kconfig"
 
 config VIRTIO_CONSOLE
 	tristate "Virtio console"
-	depends on VIRTIO && TTY
+	depends on TTY
 	select HVC_DRIVER
+	select VIRTIO
 	help
 	  Virtio console for use with lguest and other hypervisors.
 
-- 
2.28.0

